yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -203,16 +203,18 @@ func checkFileFormat() {
 
 func checkDataDirFlag() {
 	if dataDir == "" {
-		fmt.Fprintln(os.Stderr, `ERROR: required flag "data-dir" not set`)
-		os.Exit(1)
+		utils.ErrExit(`ERROR: required flag "data-dir" not set`)
 	}
 	if !utils.FileOrFolderExists(dataDir) {
-		fmt.Fprintf(os.Stderr, "Directory: %s doesn't exists!!\n", dataDir)
-		os.Exit(1)
-	} else if dataDir == "." {
+		utils.ErrExit("Directory: %s doesn't exists!!", dataDir)
+	}
+	dataDir = filepath.Clean(dataDir)
+	if filepath.Dir(dataDir) == exportDir || dataDir == filepath.Clean(exportDir) {
```

Comment:
What if exportDir is provided using relative path e.g. `./` and dataDir is provided using abs path `$PWD` . Will this code work?